src/components/appmodallogin.main.less

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.login-modal-content {
+  #login_modal_form {
+    span {
+      display: inline-block;
+      margin-bottom: 5px;
+      font-weight: bold;
+    }
+  }
+
+  #simplemodal-container {
+    background-color: #fff;
+    border: 8px solid #444;
+    padding: 12px;
+    width: auto !important;
+    height: auto !important;
+  }
+
+  .btn-auth-login {
+    display: inline-block;
+    padding: 6px 12px;
+    margin-bottom: 0;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 1.428571429;
+    text-align: center;
+    vertical-align: middle;
+    cursor: pointer;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    white-space: nowrap;
+    user-select: none;
+    padding: 10px 16px;
+    font-size: 18px;
+    line-height: 1.33;
+    border-radius: 6px;
+    border-radius: 2px;
+    box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
+    margin-bottom: 2px;
+    color: #ffffff;
+    background-color: @mainColor;
+    border-color: @mainColor;
+  }
+
+  .btn-auth-login {
+    &:focus {
+      outline: thin dotted #333;
+      outline-offset: -2px;
+    }
+
+    &:hover,
+    &:focus {
+      color: #333333;
+      text-decoration: none;
+    }
+
+    &:active,
+    &.active {
+      outline: 0;
+      box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+    }
+
+    &.disabled,
+    &[disabled],
+    fieldset[disabled] & {
+      cursor: not-allowed;
+      pointer-events: none;
+      opacity: 0.65;
+      box-shadow: none;
+    }
+
+    &:hover,
+    &:focus,
+    &:active,
+    &.active,
+    .open .dropdown-toggle& {
+      color: #ffffff;
+      background-color: rgba(0, 133, 180, 0.667);
+      border-color: rgba(0, 99, 134, 0.667);
+    }
+  }
+
+  .btn-link {
+    color: #00a3dd;
+    font-size: 14px !important;
+    font-weight: normal;
+    cursor: pointer;
+    border-radius: 0;
+    background-color: transparent;
+    box-shadow: none;
+    border-color: transparent;
+
+    &:active,
+    &[disabled],
+    fieldset[disabled] &{
+      background-color: transparent;
+      box-shadow: none;
+    }
+
+    &:hover,
+    &:focus {
+      color: #006b91;
+      text-decoration: underline;
+      background-color: transparent;
```

Comment:
Is already set at transparent without the pseudo classes, so changing it to transparent when :active doesn't make any difference. You can remove the line.

Reply:
removed